src/utils/pseudoRandomArrayItemFromString.ts

```diff
@@ -1,5 +1,8 @@
 // For a given string, will always return same item from an array.
-export default function pseudoRandomArrayItemFromString(string, array) {
+export default function pseudoRandomArrayItemFromString(
+  string: string,
+  array: any[]
+) {
```

Comment:
Using generics I think this could be something like 

```
pseudoRandomArrayItemFromString<T>(
   string: string,
   array: T[]
): T
```

since it will always return an element from `array`.